app/controllers/payments_controller.rb

```diff
@@ -11,11 +11,17 @@ def create
     else
       render json: payment.errors, status: unprocessable_entity 
     end
+
+    def show payment = Payment.find(params[:id])
+      render json: payment, status: :ok 
+    rescue ActiveRecord::RecordNotFound 
+      render json: { error: 'Payment not found' }, status: :not_found 
+    end
```

Comment:
Hi @MichelleOha - the formatting needs a fix here:

```
def show
  payment = Payment.find(params[:id]) # <- include inside of the method
  render json: payment, status: :ok 
rescue ActiveRecord::RecordNotFound 
  render json: { error: 'Payment not found' }, status: :not_found 
end
```

Reply:
Great pick up thanks Joe. Have amended this 👍